composer.json

```diff
@@ -12,7 +12,7 @@
 	"require": {
 		"php": ">=7.0",
 		"composer-plugin-api": "^1.1 || ^2.0",
-		"symfony/yaml": "^5.2"
+		"symfony/yaml": "~5.3"
```

Comment:
I think this needs to be `~5.3.0` as the tilde allows updates to the last part of the specified version iirc